config.example.js

```diff
@@ -0,0 +1,6 @@
+export const dbSettings = {
+    host: '',
+    user: '',
+    password: '',
+    database: ''
+}
```

Comment:
```suggestion
export const settings = {
  db: {
    host: "hostname",
    user: "username",
    password: "password".
    database: "database"
  }
}
```

Reply:
Optional but desirable (you may have more settings later, if this were a real world project)